content/01.abstract.md

```diff
@@ -5,6 +5,6 @@ Being invited to deliver a keynote at an international society meeting or named
 We sought to understand the extent to which such recognitions reflected the composition of their corresponding field.
 We collected keynote speaker invitations for the international meetings held by the International Society for Computational Biology as well as the names of Fellows, an honorary group within the society.
 We compared these honorees with last and corresponding author contributions in field-specific journals.
-We used multiple methods to estimate the race, ethnicity, gender, and nationality of authors and the recipients of these honors.
-To address weaknesses in existing approaches, we built a new dataset of more than 700,000 people-nationality pairs from Wikipedia and trained long short-term memory neural networks to make predictions.
+We used multiple methods to estimate the race, ethnicity, gender, and name etymology of authors and the recipients of these honors.
+To address weaknesses in existing approaches, we built a new dataset of more than 700,000 people name etymology pairs from Wikipedia and trained long short-term memory neural networks to make predictions.
```

Comment:
I think we should keep this instance as nationality. We're using the Wiki data to infer name etymology, but the raw data that we built is still name-country pairs, whether country of birth or of self-identification.

Reply:
> I think we should keep this instance as nationality.
First you need to define "nationality". It is not citizenship, it is not religion, it is not ethnicity... what is it?

Reply:
Wikipedia uses this definition: https://en.wikipedia.org/wiki/Wikipedia:Citizenship_and_nationality

Reply:
@idoerg I agree that we don't have a concrete definition of "nationality" here. Nationality from Wikipedia's card for a living person is essentially what Wikipedia curators consider to be the person's primary country. With this loose definition, there are definitely cases where the prediction for a name origin disagrees with "nationality". However, we tried to account for this issue by using prediction probabilities rather than hard assignments of origins for each name. 

I think the confusion mostly comes from NamePrism groupings, or perhaps more on the **names** of these groupings. I am not aware of any consensus on how to group countries, but we do need some groupings for our algorithm to make meaningful predictions. We would love to have your insights on how to improve these groupings or rename them to reflect accurately what we're trying to do here.

Reply:
Sorry - link should have gone here: https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Biography#Context